samples/msgext-ai-sentiment-analysis/nodejs/infra/azure.bicep

```diff
@@ -0,0 +1,44 @@
+@maxLength(20)
+@minLength(4)
+@description('Used to generate names for all resources in this file')
```

Comment:
Check if we need to push this file.

Reply:
generated default, it is in yours pr as well.